include/lmms_math.h

```diff
@@ -61,68 +61,12 @@ static inline float fraction( const float _x )
  * If the result is interpreted as a phase of an oscillator, it makes that negative phases are
  * converted to positive phases.
  */
-static inline float absFraction( const float _x )
-{
-	return( _x - ( _x >= 0.0f ? static_cast<int>( _x ) :
-						static_cast<int>( _x ) - 1 ) );
-}
-
-/*!
- * @brief Returns the fractional part of a float, a value between -1.0f and 1.0f.
- *
- * fraction( 2.3) =>  0.3
- * fraction(-2.3) => -0.3
- *
- * Note that if the return value is used as a phase of an oscillator, that the oscillator must support
- * negative phases.
- */
-static inline float fraction( const float _x )
+static inline float absFraction(const float x)
 {
-	return( _x - static_cast<int>( _x ) );
+	return x - std::floor(x);
```

Comment:
Will the behaviour change break anything tho?

Reply:
That definition was only used for the Intel compiler; this definition was used for the builds we distribute:
```cpp
static inline float absFraction( const float _x )
{
	return( _x - ( _x >= 0.0f ? static_cast<int>( _x ) :
						static_cast<int>( _x ) - 1 ) );
}
```
As for whether it will break anything, I highly doubt it, as the [0, 1) range was already returned for non-negative inputs.

Reply:
Yeah, I just figured that if it was defined like that for builds that use the Intel compiler, then the behavior *should* be the same.

Reply:
I'll leave this as is. If there's a behaviour change, we'll look later.